taskcluster/tc-build-utils.sh

```diff
@@ -340,7 +340,8 @@ do_nuget_build()
 do_deepspeech_ios_framework_build()
 {
   arch=$1
-  cp ${DS_TFDIR}/bazel-bin/native_client/libdeepspeech.so ${DS_DSDIR}/native_client/swift/libdeepspeech.so
+  unzip ${DS_TFDIR}/native_client/deepspeech_ios.zip
```

Comment:
Looks like this is not working as expected, you have tasks errors on both iOS builds: https://community-tc.services.mozilla.com/tasks/S_r3YKP2QhGddFThRV-qqg/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FS_r3YKP2QhGddFThRV-qqg%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive_backing.log#L14019

Reply:
Yep. Wrong dir it seems.